### PR TITLE
[BZ1964540] Remove ShiftStack trunking support statement 

### PR DESCRIPTION
**Pull request by maxwelldb**
As a consequence of [BZ1964540](https://bugzilla.redhat.com/show_bug.cgi?id=1964540), a statement on ports + trunking support is no longer valid. 

4.9+

Preview: https://deploy-preview-36911--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-osp

### CODE REVIEW

**Comment by maxwelldb** [comment]
Relevant code change that causes this: https://github.com/openshift/cluster-api-provider-openstack/pull/185/commits/efb5628fb08ebe2b97bbb95cf87c6e57f7601363

**Comment by maxwelldb** [comment]
@itzikb-redhat Would you be able to confirm that this change looks good? Figured it would be quick, as you QEed https://bugzilla.redhat.com/show_bug.cgi?id=1964540.

File: modules/machineset-yaml-osp-sr-iov-port-security.adoc [hunk 1]
```diff
@@ -92,8 +92,6 @@ spec:
 ====
 Trunking is enabled for ports that are created by entries in the networks and subnets lists. The name of ports that are created from these lists follow the pattern `<machine_name>-<nameSuffix>`. The `nameSuffix` field is required in port definitions.
```
Comment (MaysaMacedo) [inline modules/machineset-yaml-osp-sr-iov-port-security.adoc:93]: Sould this fragment also mention that trunks can be enabled at the Ports level?

Comment (maxwelldb) [reply modules/machineset-yaml-osp-sr-iov-port-security.adoc:93]: So, "You can enable trunking per port," "You can enable trunking per `Port` object/resource," or something else? @MaysaMacedo

Comment (MaysaMacedo) [reply modules/machineset-yaml-osp-sr-iov-port-security.adoc:93]: I guess "you can enable trunking per Port" might sound better

Comment (maxwelldb) [reply modules/machineset-yaml-osp-sr-iov-port-security.adoc:93]:
```suggestion
Trunking is enabled for ports that are created by entries in the networks and subnets lists. The name of ports that are created from these lists follow the pattern `<machine_name>-<nameSuffix>`. The `nameSuffix` field is required in port definitions.

You can enable trunking for each port.
```

@MaysaMacedo ?

And

**Review by MaysaMacedo — COMMENTED** [review]
/lgtm

File: modules/machineset-yaml-osp-sr-iov-port-security.adoc (hunk 1 above)
Comment (maxwelldb) [reply modules/machineset-yaml-osp-sr-iov-port-security.adoc:93]: Also, TODO: match in other mod.

**Review by itzikb-redhat — COMMENTED** [review]
lgtm

**Pull request merged**